en/docs/guides/access-delegation/uma.md

```diff
@@ -0,0 +1,509 @@
+# User Managed Access with WSO2 Identity Server
+
+WSO2 Identity Server (WSO2 IS) supports the [UMA 2.0](../../../references/concepts/authorization/user-managed-access) protocol, which
+allows a resource owner to easily share resources with other requesting
+parties. This tutorial demonstrates how you can use WSO2 IS as the
+authorization server to try out UMA for a sample scenario.
+
+## Introduction
+
+To demonstrate the UMA flow using WSO2 IS as the authorization server,
```

Comment:
IMO, we should make this doc more generic and not use names. 

For example, we can give the instructions to create the resource owner but not make it a scenario and give the resource owner a name since this is a guide. WDYT?

Reply:
fixed